### PR TITLE
Fix #1278

### LINKED ISSUE #1278: Add clarification on where the code for supported lines lives
We need to amend our README.md or CONTRIBUTING.md to explain that the code for v1 lives in https://github.com/fastify/fastify/tree/1.x.

### PR DESCRIPTION
**Pull request by 0xsuid**
Added description to explain that the code for v1 lives in Branch 1.x which fixes issue #1278.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by 0xsuid** [comment]
Elaborated and also added section to it @cemremengu :)

**Comment by 0xsuid** [comment]
if you want than I will update README as well @delvedor

**Comment by delvedor** [comment]
Yes please :)

**Comment by mcollina** [comment]
@0xsuid any updates on this?

**Comment by 0xsuid** [comment]
I'll give update today @mcollina

**Comment by 0xsuid** [comment]
Updated @delvedor @mcollina @cemremengu

**Comment by 0xsuid** [comment]
Just updated my commit messages.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ There are a few basic ground-rules for contributors:
 1. All integrated CI services must be green, unless an agreement has been reached in the conversation about the exception, before a pull-request can be merged.
 1. Breaking changes or significant new features must be merged by a core member. Small pull-requests, e.g. documentation typo corrections, may be merged by any member with commit access.
 
+## Fastify v1.x
+
+Code for Fastify's **v1.x** is at - [Branch 1.x](https://github.com/fastify/fastify/tree/1.x), so all Fastify 1.x related changes should be based on **`branch 1.x`**.
```
Comment (delvedor) [inline CONTRIBUTING.md]: Why the `-`?
```suggestion
Code for Fastify's **v1.x** is in [branch 1.x](https://github.com/fastify/fastify/tree/1.x), so all Fastify 1.x related changes should be based on **`branch 1.x`**.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ There are a few basic ground-rules for contributors:
 1. All integrated CI services must be green, unless an agreement has been reached in the conversation about the exception, before a pull-request can be merged.
 1. Breaking changes or significant new features must be merged by a core member. Small pull-requests, e.g. documentation typo corrections, may be merged by any member with commit access.
 
+## Fastify v1.x
```
Comment (delvedor) [inline CONTRIBUTING.md]:
```suggestion
### Fastify v1.x
```

**Comment by 0xsuid** [comment]
Both files are updated @delvedor @mcollina

**Pull request merged**